src/pages/Campaign/LeaderboardLayout.tsx

```diff
@@ -167,54 +167,59 @@ export default function LeaderboardLayout({ refreshIn }: { refreshIn: number })
             </LeaderboardTableHeaderItem>
           )}
         </LeaderboardTableHeader>
-        {searchedLeaderboard
-          .slice((currentPage - 1) * CAMPAIGN_ITEM_PER_PAGE, currentPage * CAMPAIGN_ITEM_PER_PAGE)
-          .map((data, index) => (
-            <LeaderboardTableBody
-              key={index}
-              showRewards={showRewards}
-              style={{
-                background:
-                  data.rank === 1
-                    ? 'linear-gradient(90deg, rgba(255, 204, 102, 0.25) 0%, rgba(255, 204, 102, 0) 54.69%, rgba(255, 204, 102, 0) 100%)'
-                    : data.rank === 2
-                    ? 'linear-gradient(90deg, rgba(224, 224, 224, 0.25) 0%, rgba(224, 224, 224, 0) 54.69%, rgba(224, 224, 224, 0) 100%)'
-                    : data.rank === 3
-                    ? 'linear-gradient(90deg, rgba(255, 152, 56, 0.25) 0%, rgba(255, 152, 56, 0) 54.69%, rgba(255, 152, 56, 0) 100%)'
-                    : 'transparent',
-                padding: data.rank <= 3 ? '16px 20px' : '20px',
-              }}
+        {(selectedCampaignLeaderboard?.ranking ?? []).map((data, index) => (
+          <LeaderboardTableBody
+            key={index}
+            showRewards={showRewards}
+            style={{
+              background:
+                data.rank === 1
```

Comment:
when we have multi condition like that, I think we can use:
+ switch case, 
or

```
const map = {1: xxxx, 2: yyy, 3: zzz , ...} ;  (or array)
value = map[data.rank] || defautValue 
``` 
would be better understanding and clearly.

Reply:
Sure, updated.